docs/tutorials/kernel-density.md

```diff
@@ -0,0 +1,265 @@
+```scala
+  import org.apache.spark.{SparkConf, SparkContext}
+  val conf = new SparkConf().setMaster("local").setAppName("Kernel Density")
+  val sc = new SparkContext(conf)
+```
+
+In our case, we have a collection of PointFeatures that we wish to parallelize, so we issue the command
+```scala
+  import org.apache.spark.rdd.RDD
+  val pointRdd = sc.parallelize(pts, 10)
+```
+Here, the 10 indicates that we want to distribute the data, as 10 partitions, to the available workers.  A partition is a subset of the data in an RDD that will be processed by one of the workers, enabling parallel, distributed computation, assuming the existence of a pool of workers on a set of machines.  If we exclude this value, the default parallelism will be used, which is typically the number of processors, though in this local example, it defaults to one.
+
+In order to perform the same task as in the previous section, but in parallel, we will approach the problem in much the same way: points will be assigned an extent corresponding to the extent of the associated kernel, those points will be assigned SpatialKeys based on which subtiles their kernels overlap, and each kernel will be applied to the tile corresponding to its assigned SpatialKey.  Earlier, this process was effected by a flatMap followed by a groupBy and then a map.  This very same procedure could be used here, save for the fact that groupBy, when applied to an RDD, triggers an expensive, slow, network-intensive shuffling operation which collects items with the same key on a single node in the cluster.  Instead, a fold-like operation will be used: `aggregateByKey`, which has a signature of <center style="padding-bottom: 1em">`RDD[(K, U)] => T => ((U, T) => T, (T, T) => T) => RDD[(K, T)]`.</center>
```

Comment:
This line renders quite strangely. I don't think the `<center>` works as intended.